src/main/kotlin/com/ampnet/reportservice/service/impl/TemplateDataServiceImpl.kt

```diff
@@ -146,10 +146,11 @@ class TemplateDataServiceImpl(
 
     private fun validateTransactionBelongsToUser(userWallet: WalletResponse, fromTxHash: String, toTxHash: String) {
         val txHash = userWallet.hash
-        if (txHash != fromTxHash && txHash != toTxHash) {
+        val activationData = userWallet.activationData
```

Comment:
Add a comment in code explaining this problem:
https://github.com/AMPnet/report-service/pull/24#issuecomment-731120648